integration_tests/cosmoscli.py

```diff
@@ -1751,3 +1751,12 @@ def event_query_tx_for(self, hash):
     def consensus_params(self, port, height):
         url = f"http://127.0.0.1:{port}/consensus_params?height={height}"
         return requests.get(url).json()["result"]["consensus_params"]
+
+    def send_enable(self, port):
+        url = f"http://127.0.0.1:{port}/cosmos/bank/v1beta1/params"
+        raw = requests.get(url).json()
+        return raw["params"]["send_enabled"]
```

Comment:
this and the above `consensus_params` function seems should be module level functions.